### PR TITLE
ci: enable building host tools for all platforms

### PR DESCRIPTION
**Pull request by nalajcie**
## Description

For some targets hostutils are needed to prepare target image. Ensure they are always available.

## Motivation and Context

JIRA: CI-334

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## How Has This Been Tested?

- [ ] Already covered by automatic testing.
- [ ] New test added: (add PR link here).
- [ ] Tested by hand on: (list targets here).

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing linter checks and tests passed.
- [ ] My changes generate no new compilation warnings for any of the targets.

## Special treatment

- [ ] This PR needs additional PRs to work (list the PRs, preferably in merge-order).
- [x] I will merge this PR by myself when appropriate.

### CODE REVIEW

**Review by gerard5 — DISMISSED** [review]
Looks good to me.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -13,5 +13,5 @@ on:
 
 jobs:
   call-ci:
-    uses: phoenix-rtos/phoenix-rtos-project/.github/workflows/ci-project.yml@master
+    uses: phoenix-rtos/phoenix-rtos-project/.github/workflows/ci-project.yml@nalajcie/build
```
Comment (agkaminski) [inline .github/workflows/ci.yml:16]: You might want to change this before merge, I guess

Comment (agkaminski) [reply .github/workflows/ci.yml:16]: Otherwise LGTM

Comment (nalajcie) [reply .github/workflows/ci.yml:16]: yeah, this is needed for every ci-related PR to actually test the changes - I'm always making it a separate commit to be deleted before the merge:
image

**Pull request merged**